lib/internal/modules/esm/module_job.js

```diff
@@ -2,26 +2,80 @@
 
 const {
   ArrayPrototypeJoin,
```

Comment:
```suggestion
  ArrayPrototypeFilter,
  ArrayPrototypeFind,
  ArrayPrototypeJoin,
  ArrayPrototypeMap,
```

Reply:
TIL https://github.com/nodejs/node/issues/30697#issuecomment-559763025 thanks for the suggestions.